stewards.json

```diff
@@ -3,4 +3,8 @@
         "ca": "standard_crypto/server_ca.crt",
         "endpoint": "https://steward.sommelier.standardcryptovc.com:5734"
     }
+    {
+        "ca": "teku/server_ca.crt",
+        "endpoint": "https://sommelier.teku.network:5734"
```

Comment:
This endpoint isn't accepting TCP connections on port 5734. Might be a firewall issue, config issue, or it may be down.